src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -300,7 +298,7 @@ def set_keyvault(cmd,
     connection_string = resolve_connection_string(cmd, name, connection_string)
     azconfig_client = AzconfigClient(connection_string)
 
-    keyvault_ref_value = json.dumps({"uri": secret_identifier}, ensure_ascii=False)
+    keyvault_ref_value = json.dumps({"uri": secret_identifier}, ensure_ascii=False, separators=(',', ':'))
```

Comment:
>separators [](start = 84, length = 10)

Why we care about the separator.  Are we strict about the indent and whitespace?

Reply:
This is just to make CLI consistent with portal. json dump adds extra space after : , while portal creates compact json without any whitespaces.

---
In reply to: [363912684](https://github.com/Azure/azure-cli/pull/11773#discussion_r363912684) [](ancestors = 363912684)